### PR TITLE
feat: add "enable graphql" in config-ui

### LINKED ISSUE #3528: [Feature][github_graphql] Display repo name in blueprint running status
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-devlake/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

The running status of the github_graphql plugin only displays the task ID:
 
image

However, repo and owner names are also needed, as in the github plugin:

image

Please consider adding the repo / owner names to github_graphql to make the status page more informative and consistent.

version: v0.14

### Use case

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by likyh**
# Summary

1. add enableGraphql in config-ui
2. display repo name for github graphql
3. add domain_entity for github graphql

I add a column for github connection `enable_graphql`. `Github.makePipelinePlan` will return github_graphql or github by enable_graphql.

### Does this close any open issues?
Closes #3528
Related #3529

### Screenshots

image
image

### CODE REVIEW

File: plugins/github/api/blueprint.go [hunk 1]
```diff
@@ -114,20 +114,42 @@ func makePipelinePlan(subtaskMetas []core.SubTaskMeta, scope []*core.BlueprintSc
...
+				if err != nil {
+					return nil, err
+				}
+				stage = append(stage, &core.PipelineTask{
+					Plugin:   "github_graphql",
+					Subtasks: subtasks,
+					Options:  options,
+				})
+			} else {
+				return nil, errors.Default.New("plugin github_graphql does not support SubTaskMetas")
```
Comment (keon94) [inline plugins/github/api/blueprint.go:140]: It looks like BadInput may be a better error type for this. "Default" translates to err 500 for new errors

File: config-ui/src/components/pipelines/StageTaskName.jsx [hunk 2]
```diff
@@ -61,14 +61,16 @@ const StageTaskName = (props) => {
...
-            task.plugin !== Providers.JENKINS && (
-              <>
-                @{task.options.owner}/{task.options.repo}
-              </>
-            )}
+          <strong>Task#{task.id}</strong>{' '}
+          {ProviderLabels[task?.plugin?.toUpperCase()] ||
+            task?.plugin?.toUpperCase()}{' '}
+          {(task.plugin === Providers.GITHUB ||
+            task.plugin === 'github_graphql') && (
```
Comment (mintsweet) [inline config-ui/src/components/pipelines/StageTaskName.jsx:68]: Why is this not part of the `Providers`?

File: config-ui/src/pages/configure/connections/ConnectionForm.jsx [hunk 3]
```diff
@@ -807,108 +809,152 @@ export default function ConnectionForm(props) {
...
-                inline={true}
-                labelFor='connection-ratelimit'
-                className={formGroupClassName}
-                contentClassName='formGroupContent'
+              </div>
+            </FormGroup>
+          </div>
+        )}
+        {[
+          Providers.GITHUB,
```
Comment (mintsweet) [inline config-ui/src/pages/configure/connections/ConnectionForm.jsx:879]: `Object.keys(Providers).includes()`?

File: config-ui/src/components/pipelines/StageTaskName.jsx (hunk 2 above)
Comment (likyh) [reply config-ui/src/components/pipelines/StageTaskName.jsx:68]: fixed

File: config-ui/src/pages/configure/connections/ConnectionForm.jsx (hunk 3 above)
Comment (likyh) [reply config-ui/src/pages/configure/connections/ConnectionForm.jsx:879]: No, That's not all. Just the connections with `ratelimit`

**Review by mintsweet — APPROVED** [review]
LGTM.

**Pull request merged**